tools/install-powershell.ps1

```diff
@@ -135,8 +135,21 @@ try {
         $downloadURL = "https://github.com/PowerShell/PowerShell/releases/download/v${release}/${packageName}"
         Write-Verbose "About to download package from '$downloadURL'" -Verbose
 
+        # Setting Tls to 12 to prevent the Invoke-WebRequest : The request was
+        # aborted: Could not create SSL/TLS secure channel. error.
+        $originalValue = [Net.ServicePointManager]::SecurityProtocol
+        [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12
+
         $packagePath = Join-Path -Path $tempDir -ChildPath $packageName
-        Invoke-WebRequest -Uri $downloadURL -OutFile $packagePath
+
+        try {
```

Comment:
Given that there is another `Invoke-WebRequest` call below, I suggest that we move the configuration of "SecurityProtocol" before line 93 (the outmost `try` block), and then revert to original protocol value in the `finally` block at the very end.